native/src/main/java/io/ballerina/stdlib/etl/nativeimpl/EtlCategorization.java

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright (c) 2025, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
This licence header is incorrect. It should be something with WSO2 LLC. Please check a recent PR.